src/main/java/WallClass.java

```diff
@@ -0,0 +1,13 @@
+public class WallClass {
```

Comment:
Might want to change the name of this class to `WallApp` or something, as the `Wall.java` file above would be classed as the `WallClass` essentially so it could get confusing later